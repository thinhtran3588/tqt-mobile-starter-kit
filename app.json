{
  "name": "mobile",
  "displayName": "TQT Starter Kit"
}
