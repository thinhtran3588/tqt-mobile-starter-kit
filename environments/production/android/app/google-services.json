{
  "project_info": {
    "project_number": "460761628098",
    "firebase_url": "https://tqt-apps.firebaseio.com",
    "project_id": "tqt-apps",
    "storage_bucket": "tqt-apps.appspot.com"
  },
  "client": [
    {
      "client_info": {
        "mobilesdk_app_id": "1:460761628098:android:d341644b5db941984a2a5b",
        "android_client_info": {
          "package_name": "pro.thinhtran.starterkit"
        }
      },
      "oauth_client": [
        {
          "client_id": "460761628098-kua79cavoufq51a91oluk2ft6osvfml2.apps.googleusercontent.com",
          "client_type": 1,
          "android_info": {
            "package_name": "pro.thinhtran.starterkit",
            "certificate_hash": "5e8f16062ea3cd2c4a0d547876baa6f38cabf625"
          }
        },
        {
          "client_id": "460761628098-g87fmrh182l040smb00ng28ps0svb5r7.apps.googleusercontent.com",
          "client_type": 3
        }
      ],
      "api_key": [
        {
          "current_key": "AIzaSyCabcKEv4eeDmfkEEWXZI9xt-wiYepg_rA"
        }
      ],
      "services": {
        "appinvite_service": {
          "other_platform_oauth_client": [
            {
              "client_id": "460761628098-g87fmrh182l040smb00ng28ps0svb5r7.apps.googleusercontent.com",
              "client_type": 3
            },
            {
              "client_id": "460761628098-tbm75hbrsmlgs07saish5bjjo14u3uuf.apps.googleusercontent.com",
              "client_type": 2,
              "ios_info": {
                "bundle_id": "pro.thinhtran.starterkit"
              }
            }
          ]
        }
      }
    }
  ],
  "configuration_version": "1"
}