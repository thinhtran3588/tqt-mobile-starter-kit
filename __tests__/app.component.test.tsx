import 'react-native';
import React from 'react';
import {render, toJSON, wait} from '@test-utils';
import {App} from '@app/app.component';

it('renders app correctly', async () => {
  const {baseElement, getByText} = render(<App />);
  await wait(() => getByText('paragraph'));
  expect(toJSON(baseElement)).toMatchSnapshot();
});
