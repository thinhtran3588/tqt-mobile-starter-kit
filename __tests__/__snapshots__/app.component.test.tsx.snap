// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app correctly 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <I18nextProvider
      i18n={
        Object {
          "use": [Function],
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <SafeAreaView
          style={
            Array [
              Object {
                "flex": 0,
              },
            ]
          }
        />
        <SafeAreaView
          style={
            Array [
              Object {
                "flex": 1,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 24,
                  "fontWeight": "bold",
                },
              ]
            }
          >
            H1
          </Text>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 22,
                  "fontWeight": "bold",
                },
              ]
            }
          >
            H2
          </Text>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 20,
                  "fontWeight": "bold",
                },
              ]
            }
          >
            H3
          </Text>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 18,
                  "fontWeight": "bold",
                },
              ]
            }
          >
            H4
          </Text>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 16,
                  "fontWeight": "bold",
                },
              ]
            }
          >
            H5
          </Text>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 14,
                  "fontWeight": "bold",
                },
              ]
            }
          >
            H6
          </Text>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 13,
                  "fontWeight": "bold",
                },
              ]
            }
          >
            label
          </Text>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 12,
                },
              ]
            }
          >
            paragraph
          </Text>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 12,
                },
              ]
            }
          >
            UPPERCASE
          </Text>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 12,
                },
              ]
            }
          >
            lowercase
          </Text>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "fontSize": 12,
                },
              ]
            }
          >
            settings
          </Text>
        </SafeAreaView>
      </View>
    </I18nextProvider>
  </View>
</View>
`;
